enos/modules/docker_boundary/boundary-config-rate-limit.hcl

```diff
@@ -35,8 +35,8 @@ controller {
     resources = ["host"]
     actions = ["read"]
     per = "auth-token"
-    limit = 5
-    period = "5s"
+    limit = 3
+    period = "10s"
```

Comment:
Do we need to revert this again? I believe it was changed by @moduli while we were having problems with the e2e tests. I'm going to try reverting it and see if the tests are still passing. CC @talanknight

Reply:
https://github.com/hashicorp/boundary/pull/4183 was when it was introduced.

Reply:
Opened https://github.com/hashicorp/boundary/pull/4221 to see if we can revert this.

Reply:
Looks like we still need this judging by the test results.